### PR TITLE
fix: opt-out of stubbing by passing false as stub

### PR DESCRIPTION
**Pull request by xanf**
@vue/test-utils 1 [allowed to un-stub certain component](https://github.com/vuejs/vue-test-utils/blob/dev/test/specs/mounting-options/stubs.spec.js#L299-L310) by passing `false` as stub configuration

Recover and document this behavior

### CODE REVIEW

File: tests/shallowMount.spec.ts
Comment (lmiller1990) [inline tests/shallowMount.spec.ts:104]: I wonder if we can use the `inlineSnapshot` feature from Jest here? Or do you think the string assertion is easier to read?

Thanks for all these PRs - let's do a release as soon as they are all merged up.

Comment (xanf) [reply tests/shallowMount.spec.ts:104]: I prefer to keep this as it is for now - just for consistency with style of other tests here. We could probably address it in separate clean-up MR

Comment (xanf) [reply tests/shallowMount.spec.ts:104]: A bit offtopic, regarding release. I believe I have some more (~5-10 MR coming) somewhere in next two days. I'm working (for fun) on building a prototype migration of `gitlab-ui` to Vue3 using compat build and it is surprisingly satisfying. 

With new plugin system in play I was able to mimic a huge precent of old VTU v1 behavior in pretty clean manner (for now we would like to have tests which run on both versions of VTU and)

Comment (lmiller1990) [reply tests/shallowMount.spec.ts:104]: Ok, happy to keep this as-is.

If you have 5-10 more PRs coming, we can wait until they are all merged up before doing a release.

**Review by lmiller1990 — APPROVED** [review]

File: tests/shallowMount.spec.ts
Comment (xanf) [reply tests/shallowMount.spec.ts:104]: @lmiller1990 Sure, I'll drop a comment in "last PR" for letting you know. thank you for acting super-fast

**Pull request merged**
